src/environments/environment.ts

```diff
@@ -3,7 +3,14 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  version: "/v1.0",
+  applicationName: "/ORP"
+};
+
+export const loginsetting = {
+  loginBaseUrl: "https://localhost:44339/api",
+  signInPath: "/Signin"
```

Comment:
as a practice, all verbs should be in lower case e.g. "signin"

Reply:
Done